functions.sh

```diff
@@ -3296,12 +3296,12 @@ report_statistic() {
   if [ "$1" -a "$2" -a "$3" -a "$4" -a "$5" ]; then
     REPORTSRV="$1"
 
-    STANDARDIMAGE="`ls -1 "$IMAGESPATH" |grep "$2"`"
+    STANDARDIMAGE="$(ls -1 "$IMAGESPATH" |grep "$2")"
```

Comment:
I reckon that regular bash globbing should suffice here.
```
STANDARDIMAGE=$(basename "$IMAGESPATH"/$2)
```
Note: IMAGEPATH already has a trailing "/", so the "/" here is not strictly necessary, but the original Version would have worked without the trailing slash in IMAGEPATH as well.